### ISSUE TITLE
4.0.0b版本

### ISSUE THREAD

**Issue by yangdehui** [Instagram/IGListKit#1410]
[self.collectionContext performBatchAnimated:true updates:^(id<IGListBatchContext>  _Nonnull batchContext) {
            [batchContext reloadInSectionController:self atIndexes:[NSIndexSet indexSetWithIndex:1]];
        } completion:^(BOOL finished) {
            
        }];

Refresh the current group as follows，Other groups will be affected，

**Comment by iperry90** [comment]
Hi @yangdehui, could you provide an example project showing the issue or explain the issue in more detail? Thanks!

**Comment by yangdehui** [comment]
> Hi @yangdehui, could you provide an example project showing the issue or explain the issue in more detail? Thanks!

There are two groups IGListSectionControllerA（cellA1、cellA2、cellA3） and IGListSectionControllerB（cellB1） in the project，When I refresh one cell of IGListSectionControllerA，The layoutSubviews method of cellB1 is called.

**Comment by Bruce-pac** [comment]
@yangdehui When you refresh one cell of IGListSectionControllerA, Did this cell's frame or bounds changed? If it does,I think It would affect other section, especially the section below IGListSectionControllerA.

**Comment by yangdehui** [comment]
> @yangdehui When you refresh one cell of IGListSectionControllerA, Did this cell's frame or bounds changed? If it does,I think It would affect other section, especially the section below IGListSectionControllerA.

Yes! Did this cell's frame or bounds changed, Did fold spread processing,But why does it affect IGListSectionControllerB.

**Comment by Bruce-pac** [comment]
If B below A, when A fold up, this is bound to have an affect to B's frame. When view's frame or bounds changed, it's layoutSubviews will be called.

**Comment by yangdehui** [comment]
> If B below A, when A fold up, this is bound to have an affect to B's frame. When view's frame or bounds changed, it's layoutSubviews will be called.

But that didn't happen in the official demo，emmmmm~

**Comment by Bruce-pac** [comment]
@yangdehui In the official demo, the fold effect is implemented by `self.collectionContext?.invalidateLayout(for: self)`, not reload.

**Comment by yangdehui** [comment]
> self.collectionContext

I look at the others write version of OC，the fold effect is implemented by -- self.collectionContext reloadInSectionController
